rust/protocol/src/storage/inmem.rs

```diff
@@ -287,6 +304,11 @@ pub struct InMemSignalProtocolStore {
 }
 
 impl InMemSignalProtocolStore {
+    /// Create an object with the minimal implementation of [traits::ProtocolStore], representing
+    /// the given identity `key_pair` and with the separate random seed `registration_id`.
+    ///
+    /// *TODO: make `registration_id` into a more type-safe wrapper struct! See
+    /// <https://github.com/signalapp/libsignal-client/pull/289>.*
```

Comment:
I'd rather not refer to specific PRs in the code. Even Issues we'd been staying away from but I see how that's useful for ongoing efforts.

Reply:
Ok, that's totally fine! In other codebases I was used to doing this -- it is not necessary and I will remove this and the issue itself can be the way we stay on top of those things.